bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -308,6 +333,34 @@ where
 			[GlobalConsensus(network), AccountKey20 { network: None, key: token.into() }],
 		)
 	}
+
+	fn convert_transact(
+		message_id: H256,
+		chain_id: u64,
+		sender: H160,
+		origin_kind: OriginKind,
+		fee: u128,
+		weight_at_most: Weight,
+		payload: Vec<u8>,
+	) -> (Xcm<()>, Balance) {
+		// Fee in native token of destination chain
+		let xcm_fee: Asset = (Location::here(), fee).into();
```

Comment:
We need to use native or DOT based on whether its a system parachain or not. But in future we may want to configure this per channel.

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/114/commits/1cab94c80aefdf3497d69828a22ac15bd27dbb95

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/114/commits/5a229728b1dc360c2d9939d6ca1b6ed67cfa229a configure per channel.